advanced/Scripts/piholeDebug.sh

```diff
@@ -42,19 +42,15 @@ else
 fi
 
 # shellcheck disable=SC1091
-. /etc/pihole/versions
-
-OBFUSCATED_PLACEHOLDER="<DOMAIN OBFUSCATED>"
+source /etc/pihole/versions
```

Comment:
To make it easier to identify where something is "sourced" - In case we want to get rid of all external input one day... I thought about keeping it (getting more POSIX compliant), but `shellcheck` complaint about literally 300 things and I thought one more or less doesn't matter.

Reply:
OK.

I only commented because we are trying to get the code closer to the POSIX standard and `source` is a `bash` extension.
`source` is not supported by some POSIX shells (like `dash`, for example).

Reply:
I think we can leave it as `.`. Moving further away from POSIX doesn't sound good to me. If you want to make it clear that `.` means `source` then a comment would be good to use.